include/btllib/counting_bloom_filter.hpp

```diff
@@ -216,115 +299,279 @@ class KmerCountingBloomFilter
   void insert(const std::string& seq) { insert(seq.c_str(), seq.size()); }
 
   /**
-   * Insert an element's hash values.
+   * Insert a k-mer into the filter.
    *
-   * @param hashes Integer array of hash values. Array size should equal the
-   * hash_num argument used when the Bloom filter was constructed.
+   * @param hashes Integer array of the k-mer's hash values. Array size should
+   * equal the hash_num argument used when the Bloom filter was constructed.
    */
   void insert(const uint64_t* hashes) { counting_bloom_filter.insert(hashes); }
 
   /**
-   * Insert an element's hash values.
+   * Insert a k-mer into the filter.
    *
-   * @param hashes Integer vector of hash values.
+   * @param hashes Integer vector of the k-mer's hash values.
    */
   void insert(const std::vector<uint64_t>& hashes)
   {
-    counting_bloom_filter.insert(hashes.data());
+    counting_bloom_filter.insert(hashes);
```

Comment:
My thinking was that it's more consistent to call the corresponding CBF method, which takes the same parameter -- vector of uint64_t. The `insert` method here is automatically inline, anyway, since it's declared directly in the class. The function calls should be optimized away, as it's all inline until the final insertion.

Reply:
Agreed. I'll merge your PR into `master`.